content/advanced/fine-grained-reactivity.mdx

```diff
@@ -122,11 +126,11 @@ The core functionality emerges when linking `createSignal` and `createEffect`.
 
 ### Linking signals and effects
 
-In order to introduce reactivity, the following steps are necessaary:
+In order to introduce reactivity, you need the following steps:
```

Comment:
"introducing" reactivity sounds weird to me, as if reactivity is a mystical force in the universe or something. Maybe it's better to say like "the way the reactivity system works is:" or something, and then describing it from there.